package/src/frontend/media/media.js

```diff
@@ -2,14 +2,18 @@ import {createMediaPlayer} from './createMediaPlayer';
 
 export const media = {
   players: {},
+  muteState: true,
+  mediaOff: true,
```

Comment:
I think, `mediaOff` should continue to live in `pageflow-scrolled` since it is used specifically to make scrolled section thumbnails work. Also React context is a good fit IMO since whether media should be off is really all about where a component is rendered (i.e. in the preview or in a thumbnail). It's not global in the way that `muteState` is. In the current state, `mediaOff` is never set anywhere causing autoplay to break in published entries.

Reply:
ok, I reverted back to using useMediaSettings hook for this. But still i did not find mediaOff being set anywhere?

Reply:
When a component that is rendered in a [`SectionThumbnail`](https://github.com/codevise/pageflow/blob/023fdb7c5b3b917bb94f4adafa18a31f8bd9c753/entry_types/scrolled/package/src/frontend/SectionThumbnail.js#L9), there is no provider of `MutedContext` above it. So `useMediaSettings` will fall back to the [default value of the `MutedContext` context](https://github.com/codevise/pageflow/blob/023fdb7c5b3b917bb94f4adafa18a31f8bd9c753/entry_types/scrolled/package/src/frontend/MutedContext.js#L3), which sets `mediaOff` to `true`. `Entry` [renders a provider for `MutedContext`](https://github.com/codevise/pageflow/blob/master/entry_types/scrolled/package/src/frontend/Entry.js#L63) without setting `mediaOff`, it will thus be `undefined`  for components reading it via `useMediaSettings`. It would have probably been better to explicitly set it to `false` in `Entry` to make both cases grepable.